MSAL/src/native_auth/controllers/credentials/MSALNativeAuthCredentialsController.swift

```diff
@@ -128,86 +99,21 @@ final class MSALNativeAuthCredentialsController: MSALNativeAuthTokenController,
         return []
     }
 
-    private func retrieveTokens(
+    private func retrieveIdToken(
         account: MSALAccount,
         scopes: [String],
         context: MSALNativeAuthRequestContext
-    ) -> MSALNativeAuthTokens? {
+    ) -> String? {
         do {
             let config = factory.makeMSIDConfiguration(scopes: scopes)
-            return try cacheAccessor.getTokens(account: account, configuration: config, context: context)
+            return try cacheAccessor.getIdToken(account: account, configuration: config, context: context)
         } catch {
             MSALLogger.log(
                 level: .error,
                 context: context,
-                format: "Error retrieving tokens: \(error)"
+                format: "Error retrieving IdToken"
```

Comment:
We've lost the error here as were are no longer logging it?

Reply:
Yes, we decided to not logged it because it can contain sensitive information

Reply:
Ok